### PR TITLE
Improve installLisk.sh - Closes #158

### LINKED ISSUE #158: Errors in installLisk.sh
funtctional issues:
- the root user does not have to be named "root", thus we should check whether `[ $UID -eq 0 ]`: https://github.com/LiskHQ/lisk-scripts/blob/0.3.4/downloaded/installLisk.sh#L36
- confusion between `~/.profile` and `~/.bash_profile`: https://github.com/LiskHQ/lisk-scripts/blob/0.3.4/downloaded/installLisk.sh#L65

cruft:
- Darwin-specific logic can be removed: https://github.com/LiskHQ/lisk-scripts/blob/0.3.4/downloaded/installLisk.sh#L118
- probably not needed any more: https://github.com/LiskHQ/lisk-scripts/blob/0.3.4/downloaded/installLisk.sh#L154
- can be removed: https://github.com/LiskHQ/lisk-scripts/blob/0.3.4/downloaded/installLisk.sh#L262

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by Nazgolze — DISMISSED** [review]

File: downloaded/installLisk.sh [hunk 1]
```diff
@@ -302,25 +260,17 @@ parse_option() {
...
-		if [[ "$RELEASE" != test && "$RELEASE" != "main" && "$RELEASE" != "dev" && "$RELEASE" != "beta" ]]; then
-			echo "-r <test|main|dev|beta>"
-			usage
-			exit 1
-		fi
+	if [ "$RELEASE" != test ] && [ "$RELEASE" != "main" ] && [ "$RELEASE" != "beta" ]; then
+		usage
 	fi
+
+	LISK_INSTALL="$LISK_LOCATION/lisk-$RELEASE"
```
[the remark this answers is not in the document]
Comment (fchavant) [reply downloaded/installLisk.sh]: How did you run the script? (no arguments?)

File: downloaded/installLisk.sh [hunk 2]
```diff
@@ -94,190 +80,158 @@ user_prompts() {
...
+	echo "Extracting Lisk binaries to $LISK_INSTALL"
 
+	tar xf "$LISK_VERSION" -C "$LISK_LOCATION"
 	mv "$LISK_LOCATION/$LISK_DIR" "$LISK_INSTALL"
 
 	# if user is specifying a tarball, we probably don't want to delete it
-	if [[ ! $LOCAL_TAR ]]; then
-		echo -e "\\nCleaning up downloaded files"
-		rm -f "$LISK_VERSION" "$LISK_VERSION".SHA256
+	if [[ ! -z "$LOCAL_TAR" ]]; then
```
[the remark this answers is not in the document]
Comment (fchavant) [reply downloaded/installLisk.sh]: Good catch!

File: downloaded/installLisk.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (Nazgolze) [reply downloaded/installLisk.sh]: `bash installLisk.sh upgrade -r test`

[the remark this answers is not in the document]
Comment (Nazgolze) [reply downloaded/installLisk.sh]: Also did `bash installLisk.sh upgrade -r test -f lisk-1.1.0-rc.0-Linux-x86_64.tar.gz`

**Review by Nazgolze — APPROVED** [review]

**Pull request merged**